Code/Tools/ProjectManager/Source/PythonBindings.cpp

```diff
@@ -884,6 +885,74 @@ namespace O3DE::ProjectManager
         }
     }
 
+    AZ::Outcome<GemInfo> PythonBindings::EditGem(const GemInfo& oldGemInfo, const GemInfo& newGemInfo)
+    {
+        using namespace pybind11::literals;
+
+        QStringList newList;
+
+        for(auto feature : newGemInfo.m_features)
+        {
+            if(!oldGemInfo.m_features.contains(feature))
+            {
+                //this must have been added by the user
+                newList << feature;
+            }
+        }
+
+        QStringList removeList;
+
+        for (auto feature : oldGemInfo.m_features)
+        {
+            if(!newGemInfo.m_features.contains(feature))
+            {
+                //this must have been removed by the user
+                removeList << feature;
+            }
+        }
```

Comment:
Rather than requiring old gem info and new gem info to populate new_tags and remove_tags, I believe you can just use replace_tags parameter like we do in  `UpdateProject`.  then you can remove `oldGemInfo` parameter.